### ISSUE TITLE
Competing interest not displayed

### ISSUE THREAD

**Issue by pbronka** [microsimulation/ijm#119]
**Issue**

Competing interest statement is not displayed on the website for most of the articles. However, it is visible for ijm-00202 https://microsimulation.pub/articles/00202#info . This seems to be because the following line has been included in the json file for every author: "competingInterests": "No competing interests reported". 

Is the XML for ijm-00202 different than all the other articles? I cannot see any difference, but it would seem that the information about competing interest is either not recorded in other XML files, or lost when conversion to json takes place. 

@gnott Would you please be able to let us know how the information about the competing interests has to be recorded in the XML files supplied by Exeter so it gets picked up in the conversion process and included in the json file? (A working example is the ijm-00202 above). 

Thank you!

**Comment by FAtherden-eLife** [comment]
@pbronka, if you could give some examples of articles where this information is included, but not displaying as expected that would be useful. 

As far as I know, in the XML the competing interests need to be tagged as footnotes (`<fn>`) in an `<fn-group content-type="competing-interest">`. These are then linked to from the authors `<contrib>` using `<xref>`. It looks like `ijm-00202` has capture, so it was present in the JSON. 

```xml
<fn-group content-type="competing-interest">
<label>Conflict of Interest</label>
<fn fn-type="COI-statement" id="conf1">
<p>No competing interests reported</p>
</fn>
</fn-group>
```
Looks like 00207 and 00200 used this capture too and they are rendering some content as well.

**Comment by pbronka** [comment]
@FAtherden-eLife thank you for getting back to me on this. We have previously asked Exeter to resupply all articles in order to include the competing interests statements, so the examples include all research articles starting with ijm-00198, in particular 198, 199, 201, 205, 206, 208, 210, 211, 212, 213, 214, 216, 217, 218, 219, 220. It looks like Exeter included it using the following tag: 
```
<fn-group content-type="conflict-of-conflict">
<title>Conflict of Interest</title>
<fn fn-type="COI-statement" id="conf1">
<p>No competing interests reported</p>
</fn>
</fn-group>
```

I suppose that if Exeter modified `<fn-group content-type="conflict-of-conflict">` to `<fn-group content-type="competing-interest">` and `<title>Conflict of Interest</title>` to `<label>Conflict of Interest</label>`  it should work? Would it be best if I asked for these articles to be resupplied?

**Comment by FAtherden-eLife** [comment]
@pbronka, yes I can see that 00205 which was resupplied some while ago (has presumably been converted since then), uses `<fn-group content-type="conflict-of-conflict">`, and that information isn't present.

I would suggest asking Exeter to resupply those articles using `<fn-group content-type="competing-interest">` instead, as you say. Once that's been done, they will then require reconverting.

**Comment by pbronka** [comment]
@FAtherden-eLife will do that, thank you!

**Comment by BlueReZZ** [comment]
@gnott these changes have been tested and now rolled out to more XML files so we require conversion of the following XML files please:

articles 199, 200, 201, 202, 205, 206, 208, 210, 211, 212, 213, 214, 216, 217, 218, 219, 220 

Thanks
